### PR TITLE
added timenation

### PR DESCRIPTION
**Pull request by ByRaudy**
## Type of change

- [x] Add new directory for a server
- [ ] Update directory of a server
- [ ] Documentation Update

## Checklist

- [x] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [x] I have compressed the images appropriately (e.g. on https://tinypng.com)
- [x] I have created the manifest.json with the required values

### CODE REVIEW

File: minecraft_servers/timenation/manifest.json [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  "direct_ip": "timenation.net",
+  "server_wildcards": [
+    "%.timenation.net"
+  ],
+  "supported_languages": [
+    "de",
+    "en"
+  ],
+  "social": {
+    "web": "https://www.timenation.net",
```
Comment (JakobBruening) [inline minecraft_servers/timenation/manifest.json:13]: Is the maintenance temporary? If it is planned for a longer time period, please remove the link and create a new PR as soon as the website is released.

File: minecraft_servers/timenation/manifest.json [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    "web_shop": "https://store.timenation.net",
```
Comment (JakobBruening) [inline minecraft_servers/timenation/manifest.json:14]: Website not available - Please fix or remove.

File: minecraft_servers/timenation/manifest.json [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+    "web_support": "https://timenation.net/index.php?route=/forum/topic/8-wo-kannst-du-%C3%BCberall-support-erhalten/",
```
Comment (JakobBruening) [inline minecraft_servers/timenation/manifest.json:15]: "Fix" or remove.

File: minecraft_servers/timenation/manifest.json [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+    "tiktok": "timenationnet",
+    "instagram": "timenationnetzwerk",
+    "discord": "https://discord.com/invite/8zW2Czjkqj",
+    "youtube": "https://www.youtube.com/channel/UC5IC_t6OsrULJfVVxQcV0Tg"
+  },
+  "gamemodes": {
+    "skywars": {
+      "name": "SkyWars",
+      "color": "#FF5555",
+      "url": "https://timenation.net/index.php?route=/forum/topic/10-skywars-erkl%c3%a4rung/"
```
Comment (JakobBruening) [inline minecraft_servers/timenation/manifest.json:26]: "Fix" or remove.

File: minecraft_servers/timenation/manifest.json [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+    },
+    "stagebattle": {
+      "name": "StageBattle",
+      "color": "#55FF55",
+      "url": "https://timenation.net/index.php?route=/forum/topic/11-stagebattle-erkl%c3%a4rung/"
```
Comment (JakobBruening) [inline minecraft_servers/timenation/manifest.json:31]: "Fix" or remove.

File: minecraft_servers/timenation/manifest.json [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+    },
+    "timewars": {
+      "name": "TimeWars",
+      "color": "#00AA00",
+      "url": "https://timenation.net/index.php?route=/forum/topic/9-timewars-erkl%c3%a4rung/"
```
Comment (JakobBruening) [inline minecraft_servers/timenation/manifest.json:36]: "Fix" or remove.

**Review by JakobBruening — CHANGES_REQUESTED** [review]
Additionally, please try to compress your background files.

File: minecraft_servers/timenation/manifest.json (hunk 6 above)
Comment (ByRaudy) [reply minecraft_servers/timenation/manifest.json:36]: Will be released shortly.

File: minecraft_servers/timenation/manifest.json (hunk 5 above)
Comment (ByRaudy) [reply minecraft_servers/timenation/manifest.json:31]: Will be released shortly.

File: minecraft_servers/timenation/manifest.json (hunk 3 above)
Comment (ByRaudy) [reply minecraft_servers/timenation/manifest.json:15]: Will be released shortly.

File: minecraft_servers/timenation/manifest.json (hunk 2 above)
Comment (ByRaudy) [reply minecraft_servers/timenation/manifest.json:14]: Will be released shortly.

File: minecraft_servers/timenation/manifest.json (hunk 1 above)
Comment (ByRaudy) [reply minecraft_servers/timenation/manifest.json:13]: No its not. We will Release the Website shortly.

**Comment by ByRaudy** [comment]
> Additionally, please try to compress your background files.

I have do it?

**Pull request closed without merging**